the-collab-lab/tcl-38-smart-shopping-list
# Add test for ItemList component

## Description

This PR adds a test for the Item List component, using mocks of the token context and custom firebase hook.

## Related Issue

Closes #55 

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|   | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|  ✓  | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |


## Testing Steps / QA Criteria

Pull down branch and run the <npm run test> command. ItemList.test.js should pass.
